mathics/builtin/logic.py

```diff
@@ -118,6 +125,38 @@ class Not(PrefixOperator):
     }
 
 
+class Nand(Builtin):
+    """
+    <dl>
+    <dt>'Nand[$expr1$, $expr2$, ...]'
+    <dt>'$expr1$ && $expr2$ && ...'
+        <dd> Implements the logical `NOR` function.  The same that Not[And[$expr1$, $expr2$, ...]]
```

Comment:
NOR seems to be a typo.

Reply:
Thanks - should be fixed now in 369f6d18